### PR TITLE
add attachedcluater e2e test

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What this PR does / why we need it**:
add attachedcluater e2e test
**Which issue(s) this PR fixes**:
a part of #436

### CODE REVIEW

File: test/install.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script installs Kurator cluster-operater and fleet-manager.
+
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+MAIN_KUBECONFIG=${MAIN_KUBECONFIG:-"${KUBECONFIG_PATH}/kurator-host.config"}
+export KUBECONFIG=${MAIN_KUBECONFIG}
+VERSION=${VERSION:-"0.6.0"}
```
Comment (hzxuzhonghu) [inline test/install.sh:12]: We should build from code

File: hack/e2e-test/install.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  create: false
+EOF
+
+sleep 5s
+
+helm repo add kurator https://kurator-dev.github.io/helm-charts
+helm repo update
+sleep 5s
+
+helm install --create-namespace  kurator-cluster-operator kurator/cluster-operator --version=${VERSION} -n kurator-system
```
Comment (hzxuzhonghu) [inline hack/e2e-test/install.sh:38]: we cannot install via helm, because this version is not released yet

**Review by hzxuzhonghu — CHANGES_REQUESTED** [review]

File: test/install.sh (hunk 1 above)
Comment (LiZhenCheng9527) [reply test/install.sh]: I get it. I need to pull the latest code commits, and use the built images and chart packages to install.

File: hack/e2e-test/install.sh (hunk 2 above)
Comment (LiZhenCheng9527) [reply hack/e2e-test/install.sh:38]: I need to pull the latest code commits, and use the built images and chart packages to install.

File: hack/e2e-test/fleet-clusters/attachedcluster.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```
Comment (hzxuzhonghu) [inline hack/e2e-test/fleet-clusters/attachedcluster.sh:1]: nit: we should provife e2e case in go, could improve later

File: hack/e2e-test/install.sh (hunk 2 above)
Comment (hzxuzhonghu) [reply hack/e2e-test/install.sh:38]: nope, the code in here, you should not use the helmchart from helm registry

**Comment by hzxuzhonghu** [comment]
/close 


close first, seems break prow tide

**Pull request closed without merging**